rustup.yaml

```diff
@@ -0,0 +1,41 @@
+package:
+  name: rustup
+  version: 1.25.1
+  epoch: 0
+  description: "rustup is an installer for
+the systems programming language Rust"
```

Comment:
This seems to have worked, but I think I'd rather get rid of the new line.

Reply:
Done!